src/Mono.Android/Mono.Android.targets

```diff
@@ -128,6 +129,7 @@
         SourceFiles="$(IntermediateOutputPath)__dex\classes.dex"
         DestinationFiles="$(OutputPath)mono.android.dex"
     />
+    <Touch Files="$(OutputPath)mono.android.dex" />
```

Comment:
Losing changes here is fine; i just want to speed up a specific local rebuild, and I encountered this target constantly re-running.

If you'd prefer, I can just kill this target and integrate the appropriate changes from PR #514?